### PR TITLE
LRQA-69808 Forms#AddFormInGlobal Test Fix

### PR DESCRIPTION
**Pull request by leticiachiu**
Related Issue: https://issues.liferay.com/browse/LRQA-69808

### CODE REVIEW

**Comment by liferay-forms** [comment]
ci:test:sf

**Comment by liferay-forms** [comment]
Hi, you don’t need to run the CI commands anymore, I’ll do it for you and let you know if any issue happens.

**Comment by liferay-forms** [comment]
ci:test:relevant

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/forms/Forms.testcase [hunk 1]
```diff
@@ -46,7 +46,11 @@ definition {
 
 		PageTemplatesAdmin.openPageTemplatesAdmin(siteURLKey = "global");
 
-		FormsAdmin.addForm();
```
Comment (natocesarrego) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/forms/Forms.testcase:49]: @leticiachiu, can we add a parameter, `groupName` for example, to make this macro support `Guest` and `Global` groups?

This way we'll keep this method faster than previously, what do you think?

**Comment by leticiachiu** [comment]
@natocesarrego Done! please check if you agree with my alterations :)

**Comment by natocesarrego** [comment]
Makes sense to me, @leticiachiu! Great job! :tada:

**Comment by natocesarrego** [comment]
ci:forward

**Comment by liferay-forms** [comment]
ci:test:relevant

**Pull request closed without merging**